### PR TITLE
enh: allow additional dcm2niix configuration

### LINKED ISSUE #206: Dcm2niix options and Neurodocker line 4
Hello, 

I have a few questions while running Heudiconv: 

1. Is there a way to pass an option to dcm2niix while running heudiconv? For example to force stacking.

2. I am noticing after the conversation and while the embedder is running, neurodocker gives an error in the form of: /neurodocker/startup.sh: line 4: 9 Killed   $* . The number (9 in this case) may fluctuate, but the same error is given every time. I have provided a screenshot of the entire readout with the heudiconv call and the associated heuristic file. 

heuristic_connect
heudiconv_neurodocker_line4

Thank you!

### PR DESCRIPTION
**Pull request by mgxd**
Fixes #206 

* allow passing of an interface config through the use of `Nipype`'s `BaseInterface.__init__(from_file)`

### CODE REVIEW

**Comment by mgxd** [comment]
no objections?

**Comment by yarikoptic** [comment]
what is the usecase you are trying to fulfill? i.e. what custom option needs to be passed which should not be a part of the heuristic (as --bids should become)?

**Comment by yarikoptic** [comment]
What is your usecase, it what settings to change in dcm2niix?

**Comment by yarikoptic** [comment]
hm, one use case just came up: https://github.com/rordenlab/dcm2niix/issues/198

File: heudiconv/convert.py [hunk 1]
```diff
@@ -394,9 +413,11 @@ def nipype_convert(item_dicoms, prefix, with_prov, bids, tmpdir):
 
     item_dicoms = list(map(op.abspath, item_dicoms)) # absolute paths
 
-    dicom_dir = op.dirname(item_dicoms[0]) if item_dicoms else None
+    fromfile = dcmconfig if dcmconfig else None
+    if fromfile:
+        lgr.info("Using custom config file %s", fromfile)
 
-    convertnode = Node(Dcm2niix(), name='convert')
+    convertnode = Node(Dcm2niix(from_file=fromfile), name='convert')
```
[the remark this answers is not in the document]
Comment (satra) [reply heudiconv/convert.py]: `from_file` is s generic keyword for all interfaces to load configuration settings from a json file.

**Pull request merged**